src/config/constants.js

```diff
@@ -11,3 +11,5 @@ export const COLORS = [
   "#f4d0b5",
   "#a5673f",
 ];
+export const DESKTOP_MIN_WIDTH = 992;
```

Comment:
This value is defined in semantic-ui, but I couldn't find anywhere where we can extract in from